databricks/koalas/series.py

```diff
@@ -1870,6 +1870,70 @@ def drop(
         else:
             raise ValueError("Need to specify at least one of 'labels' or 'index'")
 
+    def filter(self, items=None, like=None, regex=None, axis=0):
```

Comment:
Can you move this into `_Frame` and let it share one implementation?

Reply:
@HyukjinKwon You mean, I should also move `DataFrame.filter()` currently implemented in the `DataFrame`  into `generic._Frame` and let it share one implementation?

Reply:
@beobest2 Correct.